test/conversion.test.js

```diff
@@ -27,6 +27,16 @@ describe('Curl converter should', function() {
 		});
 	});
 
+	it('[Github: #1]: not throw an error for having $ before method', function (done) {
```

Comment:
This isn't only for the method. Either we test only for method, or we mention method and URL in the test name.